README.rst

```diff
@@ -26,10 +26,10 @@ The documentation contains also documentation for our clients, PySparkling and R
 Download Binaries
 ~~~~~~~~~~~~~~~~~
 
-- `Latest version for Spark 2.4 <http://h2o-release.s3.amazonaws.com/sparkling-water/rel-2.4/latest.html>`__
-- `Latest version for Spark 2.3 <http://h2o-release.s3.amazonaws.com/sparkling-water/rel-2.3/latest.html>`__
-- `Latest version for Spark 2.2 <http://h2o-release.s3.amazonaws.com/sparkling-water/rel-2.2/latest.html>`__
-- `Latest version for Spark 2.1 <http://h2o-release.s3.amazonaws.com/sparkling-water/rel-2.1/latest.html>`__
+- `Latest version for Spark 2.4 <http://h2o-release.s3.amazonaws.com/sparkling-water/spark-2.4/latest.html>`__
```

Comment:
we were pointing to old location, but location has changed since the introduction of single version